src/TLuaInterpreter.cpp

```diff
@@ -11127,7 +11127,10 @@ void TLuaInterpreter::msdp2Lua(char* src, int srclen)
                 }
                 if (last == MSDP_VAL) {
                     no_array_marker_bug = true;
-                    script.append(R"(",)");
+                    script.append(R"(")");
```

Comment:
To clarify: this was appending `",`  and now it is appending `"`, surely in this case we can just use the single character form: `script.append('"');`? {I'm not sure that the new C++11/14 literals are any easier to read, or maybe I just need to get my eye in... 🙁 }

_⚠️ 🌏 __I18n__ Oh, `string` is a `QString` isn't it - so these will need to be gone through and changed to `script.append(QLatin1String(`...`)` if ... is more than a single character anyhow - there does not necessarily seem to be a `script.append(QLatin1Char(`...`)` though! ⚠️_